packages/theming/apple-theme/src/appleTheme.colors.macos.ts

```diff
@@ -0,0 +1,223 @@
+    buttonText: appleSemantic.controlTextColor,
+    buttonTextHovered: appleSemantic.controlTextColor,
+    buttonTextChecked: appleSemantic.controlTextColor,
+    buttonTextCheckedHovered: appleSemantic.controlTextColor,
+    buttonTextPressed: appleSemantic.controlTextColor,
+    buttonTextDisabled: appleSemantic.controlTextColor,
+    buttonBorderDisabled: 'transparent',
+    buttonBorderFocused: 'transparent',
+
+    primaryButtonBackground: fluentApple.communicationBlue,
+    primaryButtonBackgroundHovered: fluentApple.communicationBlue,
+    primaryButtonBackgroundPressed: fluentApple.communicationBlueTint10,
+    primaryButtonBackgroundDisabled: fluentApple.gray100,
+    primaryButtonBorder: 'transparent',
+    primaryButtonBorderFocused: 'transparent',
+    primaryButtonText: appleSemantic.controlTextColor,
+    primaryButtonTextHovered: appleSemantic.controlTextColor,
+    primaryButtonTextPressed: appleSemantic.controlTextColor,
+    primaryButtonTextDisabled: appleSemantic.disabledControlTextColor,
+
+    accentButtonBackground: fluentApple.communicationBlueTint10,
+    accentButtonText: appleSemantic.controlTextColor,
+
+    //TODO Differentiate Menu and List?
+    menuBackground: appleSemantic.alternatingEvenContentBackgroundColor,
+    menuDivider: appleSemantic.separatorColor,
+    menuIcon: appleSemantic.textColor,
+    menuHeader: appleSemantic.headerTextColor,
+    menuItemBackgroundHovered: appleSemantic.alternatingOddContentBackgroundColor,
+    menuItemBackgroundPressed: appleSemantic.selectedContentBackgroundColor,
+    menuItemText: appleSemantic.textColor,
+    menuItemTextHovered: appleSemantic.textColor,
+
+    listBackground: appleSemantic.alternatingOddContentBackgroundColor,
+    listText: appleSemantic.textColor,
+    listItemBackgroundHovered: appleSemantic.alternatingOddContentBackgroundColor,
+    listItemBackgroundChecked: appleSemantic.selectedContentBackgroundColor,
+    listItemBackgroundCheckedHovered: appleSemantic.selectedContentBackgroundColor,
+
+    listHeaderBackgroundHovered: appleSemantic.headerTextColor,
+    listHeaderBackgroundPressed: appleSemantic.headerTextColor,
+
+    // TODO ActionLink vs Link?
+    actionLink: appleSemantic.linkColor,
+    actionLinkHovered: appleSemantic.linkColor,
+    link: appleSemantic.linkColor,
+    linkHovered: appleSemantic.linkColor,
+    linkPressed: appleSemantic.selectedControlColor,
+
+    /* ControlColorTokens */
+
+    // Default values without any style
+    buttonBackground: fluentApple.communicationBlue,
+    buttonBorder: 'transparent',
+    buttonContent: appleSemantic.controlTextColor,
+    buttonIcon: appleSemantic.controlTextColor,
+
+    // macOS Button has no hover effect
+    buttonHoveredBackground: fluentApple.communicationBlue,
+    buttonHoveredBorder: 'transparent',
+    buttonHoveredContent: appleSemantic.controlTextColor,
+    buttonHoveredIcon: appleSemantic.controlTextColor,
+
+    // TODO what does focus even mean on macOS?
```

Comment:
Like via keyboard focus?

Reply:
I think so. I think Focus does something different on windows than macOS. On macOS (and I assume iOS..) we add the blue rectangle around the control, but otherwise the control looks at rest. I think on windows it takes on a new state like hovered or pressed.

I made the assumption that all the "Focused" state tokens should match rest. Again would love to hear opinions from others like @HeyImChris or @harrieshin or designers or such

Reply:
Yep. I assuemd that the tokens would match their state at rest, and the system would just add the keyboard focus rectangle.

Reply:
I think the way you solved this makes sense.